packages/browser-wallet/src/shared/storage/transactions.ts

```diff
@@ -0,0 +1,22 @@
+import { BrowserWalletAccountTransaction } from '@popup/shared/utils/transaction-history-types';
+import { makeStorageAccessor } from './access';
+import { ChromeStorageKey, StoredBrowserWalletTransaction } from './types';
+
+export const pendingTransactions = makeStorageAccessor<StoredBrowserWalletTransaction[]>(
+    'local',
+    ChromeStorageKey.PendingTransactions
+);
+
+export const fromStoredTransaction = (t: StoredBrowserWalletTransaction): BrowserWalletAccountTransaction => ({
+    ...t,
+    time: BigInt(t.time),
+    amount: BigInt(t.amount),
+    cost: t.cost === undefined ? undefined : BigInt(t.cost),
+});
+
+export const toStoredTransaction = (t: BrowserWalletAccountTransaction): StoredBrowserWalletTransaction => ({
```

Comment:
When serializing big integers we usually use our own JSONHelper to stringify/parse.

Reply:
Makes sense, I'll adapt.